app/brush.ts

```diff
@@ -319,13 +321,11 @@ export function cycleBrushHead() {
 }
 
 export function decrementBrush() {
-  rawBrushSize = Math.max(brushSize - 1, 0);
-  brushSize = Math.max(rawBrushSize, 1);
+  brushSize = clamp(Math.max(brushSize - 1), 0, MAX_BRUSH_SIZE); 
```

Comment:
The use of `Math.max` doesn't seem to do anything here as you are only calling it with one variable?